silence/silence.go

```diff
@@ -717,7 +717,12 @@ func (s *Silences) Merge(b []byte) error {
 	s.mtx.Lock()
 	defer s.mtx.Unlock()
 
+	now := s.now()
+
 	for _, e := range st {
+		if e.ExpiresAt.After(now) {
```

Comment:
I am confused, so if the silence has not yet expired, continue? Shouldn't this be:

```golang
 if e.ExpiresAt.Before(now) { 
```

Reply:
I had the same remark :-)

Reply:
Also are notification logs affected too?

Reply:
I went for using `time.Before()`. It makes it easier to read in my mind.